src/__tests__/App.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import store from '../redux/configureStore';
+import App from '../components/App';
+
```

Comment:
Please add integration tests for `data fetching` and `Redux Store connection` using `React Testing Library`.
The app has snapshot tests only so please add unit tests for components.